deployment/sandbox-v2/docs/idacert.md

```diff
@@ -20,212 +21,126 @@
 	  "version": "string"
 	}
 	```
-  * 2. Get the ROOT certificate data from the below URL, Copy it and use it for certificate exchange in the next step.
-	```https://minibox.mosip.net/idauthentication/v1/internal/getCertificate?applicationId=ROOT```
-
-  * 3. Upload ROOT certificate from the above request in the below SWAGGER URL:- ```https://minibox.mosip.net/v1/partnermanager/swagger-ui.html#/Partner%20Service%20Controller/uploadCACertificateUsingPOST``` Partner_Service_Controller --> /partners/certificate/ca/upload --> with below request
+  * 2. Get the certificate data from the below URL.The used application id = IDA ad reference id =CRED_SERVICE  
+	```https://minibox.mosip.net/idauthentication/v1/internal/swagger-ui/index.html?configUrl=/idauthentication/v1/internal/v3/api-docs/swagger-config#/keymanager/getCertificate```
+       make sure to copy the certificate value in the response from the above request.
+       
+  * 3. Upload the copied certificate from step-2 through the below SWAGGER URL:-
+       ```https://minibox.mosip.net/v1/keymanager/swagger-ui/index.html?configUrl=/v1/keymanager/v3/api-docs/swagger-config#/keymanager/uploadOtherDomainCertificate``` 
+      
+       Request body:
 	```
 	{
+     "id": "string",
+     "version": "string",
+     "requesttime": "2023-05-30T04:36:36.006Z"
+     "metadata": {},
+     "request": {
+     "applicationId": "IDA",
+     "referenceId": "PUBLIC_KEY",
+     "certificateData": "{copied certificate from step-II}"
+  }
+  }
+	```
+ 
+ P.S : The process of uploading IDA Zero-Knowledge certificates should only take place once per environment.
+ 
+  B)for IDA mpartner-default-auth partner certificate exchange, below are the steps
```

Comment:
For IDA mpartner-default-auth partner certificate exchange, below are the steps: